ModelBase/src/model/TreeManager.h

```diff
@@ -490,6 +506,9 @@ inline Node* TreeManager::root(){ return root_; }
 inline QString TreeManager::name() { return name_; }
 inline void TreeManager::setName(const QString& name) { name_ = name; }
 
+inline QString TreeManager::managerName() { return managerName_; }
```

Comment:
yes, `name` is also not ideal, but it's also not that bad, since a manager's name is supposed to be the name of the top-level root node.